src/instana/instrumentation/flask/common.py

```diff
@@ -4,49 +4,71 @@
 
 import wrapt
 import flask
-import opentracing
-import opentracing.ext.tags as ext
+from importlib.metadata import version
+from typing import Callable, Tuple, Dict, Any, TYPE_CHECKING, Union
 
-from ...log import logger
-from ...singletons import tracer, agent
+from opentelemetry.semconv.trace import SpanAttributes as ext
+
+from instana.log import logger
+from instana.singletons import tracer, agent
+from instana.propagators.format import Format
+from instana.instrumentation.flask import signals_available
+
+
+if TYPE_CHECKING:
+    from instana.span.span import InstanaSpan
+    from werkzeug.exceptions import HTTPException
+    from flask.typing import ResponseReturnValue
+
+    if signals_available:
+        from werkzeug.datastructures.headers import Headers
+    else:
+        from werkzeug.datastructures import Headers
 
 
 @wrapt.patch_function_wrapper('flask', 'templating._render')
-def render_with_instana(wrapped, instance, argv, kwargs):
+def render_with_instana(
+    wrapped: Callable[..., str], instance: Any, argv: Tuple, kwargs: Dict
```

Comment:
The callable is templating._render. If I try to print he instance, it is `None`. Figured out the values for argv and kwrags
```
    argv: Tuple[flask.app.Flask, "Template", Dict[str, Any]],
    kwargs: Dict[str, Any],
```

Reply:
As discussed, using `object` instead of `Any` for `instance`.
```suggestion
    wrapped: Callable[..., str],
    instance: object,
    argv: Tuple[flask.app.Flask, "Template", Dict[str, Any]],
    kwargs: Dict[str, Any],
```